.travis.yml

```diff
@@ -29,7 +29,10 @@ before_install:
   - conda info -a
 
 install:
-  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION $CONDA_PACKAGES
+  - conda install anaconda=2.4
+  #- conda create --name testoceanmodes --file oceanmodes-exported-packages.txt
+  #- source activate testoceanmodes
+  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION $CONDA_PACKAGES --file oceanmodes-exported-packages.txt
```

Comment:
We should try to simplify the exported-packages.txt file so it contains only the packages necessary to run the `oceanmodes` (scipy and numpy) PLUS their dependencies. That will help us get to the bottom of the problem.